src/simple-yenc.js

```diff
@@ -36,6 +36,10 @@ const decode = (string) => {
       continue;
     }
 
+    if (byte > 255) {
+      byte = 128 + [8364,,8218,402,8222,8230,8224,8225,710,8240,352,8249,338,,381,,,8216,8217,8220,8221,8226,8211,8212,732,8482,353,8250,339,,382,376].indexOf(byte)
```

Comment:
Which character code set are the numbers representing in the array? I'm a bit concerned that these values may be different depending on the character encoding of the source string.

Reply:
html automatically changes byte values 128-160 the these unicode code points. i am expressing them as integers (and not strings) so they can work with custom html charsets. see here: https://html.spec.whatwg.org/multipage/parsing.html#table-charref-overrides